Keyname overhaul (#3115)
ElektraInitiative/libelektra: doc/decisions/bootstrap.md

Currently the default backend (default.ecf) will also be used for bootstrapping. There are two problems with this approach:

> 1. Thus the default backend first will be read with parentKey `system:/elektra` and later with parentKey `system`, it needs to store absolute paths and thus won't work with the current INI plugin

Reviewer: ```suggestion
1. Thus the default backend first will be read with parentKey `system:/elektra` and later with parentKey `system:/`, it needs to store absolute paths and thus won't work with most of the plugins (except dump).
```

## Considered Alternatives

> - Implement a hack so that `system:/elektra` is actually read as `system`. (Will not solve problem 2.)

Reviewer: ```suggestion
- Implement a hack so that `system:/elektra` is actually read as `system:/`. (Will not solve problem 2.)
```
